src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -825,28 +825,24 @@ private static ProjectTargetInstance ReadNewTargetElement(ProjectTargetElement t
                 EvaluateItemGroupElement(itemGroupElement, lazyEvaluator);
             }
 
-            if (lazyEvaluator != null)
+            // Tell the lazy evaluator to compute the items and add them to _data
```

Comment:
The lazy evaluator was previously introduced and the null / local instantiation are a sort of a poor man's escape hatch to revert back to the old behaviour if need be. At some point in the future (after the next VS release maybe?) we should remove the escape hatch and delete all the downstream code.